osu.Game/Scoring/Legacy/ScoreInfoExtensions.cs

```diff
@@ -68,6 +68,22 @@ private static long convertStandardisedToClassic(int rulesetId, long standardise
             }
         }
 
+        public static double GetOsuClassicScoreMultiplier(this ScoreProcessor scoreProcessor)
+        {
+            int maxBasicJudgements = scoreProcessor.MaximumStatistics
+                                       .Where(k => k.Key.IsBasic())
+                                       .Select(k => k.Value)
+                                       .DefaultIfEmpty(0)
+                                       .Sum();
+
+            return getOsuClassicScoreMultiplier(maxBasicJudgements);
+        }
+
+        private static double getOsuClassicScoreMultiplier(int objectCount)
+        {
+            return (Math.Pow(objectCount, 2) * 32.57 + 100000) / ScoreProcessor.MAX_SCORE;
```

Comment:
what on earth is this formula

Reply:
comes from https://github.com/ppy/osu/pull/24924, yeah it's ridiculous. was mostly intended to be used as providing a fake large number to trick stable users into believing they're still on stable scoring. but I do agree that using this fake large number on spinners feels pretty dreadful because of how random the numbers are, which is why I tagged you in here to ask if we want to even try any of this